packages/atomic/src/locales.json

```diff
@@ -1212,6 +1212,14 @@
     "en": "Apply custom start and end dates for the {{label}} facet",
     "fr": "Appliquer les dates de début et de fin manuelles pour la facette {{label}}"
   },
+  "past-minute": {
```

Comment:
Why is this needed now ?

Reply:
I forgot to mention that.

While testing with typings, I noticed that this was missing. It's not strictly needed for the PR.